ada/testsuite/tests/name_resolution/pragma_assert/p.adb

```diff
@@ -1,8 +1,15 @@
-   procedure P is
-      function "-"(I: Integer) return Integer is (I*I);
+procedure P is
+   function "-" (I: Integer) return Integer is (I*I);
```

Comment:
Look at you being all fancy about Ada coding style

Reply:
Nice :) We just miss spaces around the `*` operator :-)

Reply:
Ha, too late. This will be for the next person who touches this test!